tcmur_cmd_handler.c

```diff
@@ -1802,7 +1806,7 @@ static void handle_caw_read_cbk(struct tcmu_device *dev,
 	}
 
 	/* perform write */
-	tcmu_seek_in_iovec(origcmd->iovec, state->requested);
+	tcmu_seek_in_cmd_iovec(origcmd, state->requested)
```

Comment:
Missing ";".

Reply:
I have fixed it. Thanks for your review.